packages/contracts-bedrock/contracts/test/CommonTest.t.sol

```diff
@@ -208,8 +216,15 @@ contract Messenger_Initializer is L2OutputOracle_Initializer {
         super.setUp();
 
         // Deploy the OptimismPortal
-        op = new OptimismPortal(oracle, 7 days);
+        op = new OptimismPortal({
+            _l2Oracle: oracle,
+            _guardian: guardian,
+            _finalizationPeriodSeconds: 7 days,
+            _paused: false
+        });
         vm.label(address(op), "OptimismPortal");
+        vm.prank(op.GUARDIAN());
+        op.unpause();
```

Comment:
Portal should already be unpaused, was instantiated w/ `_paused: false` 4 lines above.

Reply:
Ah good catch